tests/test_eload_brokering.py

```diff
@@ -96,10 +96,8 @@ def test_collect_brokering_workflow_results(self):
         })
         self.eload._collect_brokering_prep_results(tmp_dir)
         vcf_file1 = os.path.join(self.eload.eload_dir, '18_brokering/ena/vcf_file1.vcf.gz')
-        vcf_file1_index = os.path.join(self.eload.eload_dir, '18_brokering/ena/vcf_file1.vcf.gz.tbi')
         vcf_file1_csi = os.path.join(self.eload.eload_dir, '18_brokering/ena/vcf_file1.vcf.gz.csi')
         assert os.path.isfile(vcf_file1)
-        assert os.path.isfile(vcf_file1_index)
```

Comment:
1. Probably not necessary but we might as well `assert os.path.isfile(vcf_file1_csi)` here instead.
2. On line 88 above, remove the `.tbi` and `.tbi.md5` files from the list of temp files to create.